pkg/operator/main.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-var operatorLogger = logging.GetOperatorLogger()
+var operatorLogger = logging.GetLogger()
```

Comment:
@vishalbollu might want to check if this is okay

Reply:
LGTM